pkg/k8shandler/logstore.go

```diff
@@ -39,7 +39,7 @@ func CreateOrUpdateLogStore(cluster *logging.ClusterLogging) (err error) {
 		retryErr := retry.RetryOnConflict(retry.DefaultRetry, func() error {
 			if exists, cluster := utils.DoesClusterLoggingExist(cluster); exists {
 				if !reflect.DeepEqual(elasticsearchStatus, cluster.Status.LogStore.ElasticsearchStatus) {
-					logrus.Infof("Updating status of Elasticsearch")
+					logrus.Info("Updating status of Elasticsearch")
```

Comment:
I think it is useful to print this, but not more than once if looping in the RetryOnConflict